### PR TITLE
[cherry-pick](branch-2.1) Pick "[Fix](group commit) Fix table not found fault when disable group commit (#39731)"

### PR DESCRIPTION
**Pull request by Yukang-Lian**
## Proposed changes

Pick #39731

### CODE REVIEW

**Comment by Yukang-Lian** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/LoadAction.java [hunk 1]
```diff
@@ -337,7 +349,18 @@ private TNetworkAddress selectRedirectBackend(HttpServletRequest request, boolea
             Backend backend = Env.getCurrentSystemInfo().getBackend(debugBackendId);
             return new TNetworkAddress(backend.getHost(), backend.getHttpPort());
         }
-        return selectLocalRedirectBackend(groupCommit, request, tableId);
+        if (Config.isCloudMode()) {
+            String cloudClusterName = getCloudClusterName(request);
```
Comment (mymeiyi) [inline fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/LoadAction.java:353]: unused

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by yiguolei** [comment]
run buildall

**Comment by Yukang-Lian** [comment]
run buildall

**Pull request merged**